### PR TITLE
Fixing branch for Cordova plugin

### PR DESCRIPTION
**Pull request by bhariharan**
It was incorrectly pointing to ```unstable```.

### CODE REVIEW

File: node/forcedroid.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ var fs = require('fs');
 var cordovaHelper = require('../external/shared/node/cordovaHelper');
 var miscUtils = require('../external/shared/node/utils');
 
-var version = '3.1.0';
```
Comment (wmathurin) [inline node/forcedroid.js]: Do we need to bump up the version number?

Comment (bhariharan) [inline node/forcedroid.js]: This is only the ```npm``` package version number. We could do a force override, but that's not preferable (I don't know what will happen to the users who have already downloaded ```3.1.0```).

Comment (wmathurin) [inline node/forcedroid.js]: Sounds good.

**Comment by wmathurin** [comment]
Looks fine.

**Pull request merged**